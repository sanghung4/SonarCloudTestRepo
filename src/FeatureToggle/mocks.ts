import { Feature } from 'generated/graphql';

export const mockFeatures: Feature[] = [
  { id: '1', isEnabled: false, name: 'noooo' },
  { id: '2', isEnabled: true, name: 'fine' },
  { id: '3', isEnabled: true, name: 'test' }
];
