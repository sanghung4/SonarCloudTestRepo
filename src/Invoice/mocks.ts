import { GetInvoiceDocument } from 'generated/graphql';

export const successMincron = {
  request: {
    query: GetInvoiceDocument,
    variables: {
      accountId: '123456',
      invoiceNumber: '5460026'
    }
  },
  result: {
    data: {
      invoice: {
        invoiceNumber: '5460026',
        status: 'Open',
        terms: 'NET 30 DAYS',
        customerPo: '40251/13325',
        invoiceDate: '10/28/2020',
        dueDate: '10/28/2020',
        originalAmt: '',
        openBalance: '',
        age: '',
        jobNumber: 'AVLNEP7',
        jobName: 'NEP 7 ASHEVILLE',
        address: {
          streetLineOne: '911 BLUEBERRY WAY',
          streetLineTwo: '',
          streetLineThree: '',
          city: 'NORTHLAKE',
          state: 'TX',
          postalCode: '76226',
          country: ''
        },
        shipDate: '11/06/2020',
        deliveryMethod: 'Our Truck',
        subtotal: '3184.28',
        tax: '262.76',
        otherCharges: '0.0',
        paidToDate: '1000.00',
        invoiceItems: [
          {
            id: '1738433F4-91A7-4637-8C60-B4DF59A060C1',
            brand: 'Advanced Drainage Systems, Inc.',
            name: '120 x 7.5, 100 Sq Yd, Geosynthetic, Double, Photodegradable, Erosion Control Blanket',
            partNumber: '8X1125STRAWDN',
            mfr: '00S2TT',
            thumb: 'null',
            price: '43.59',
            qty: {
              quantityOrdered: 1,
              quantityShipped: 1
            }
          }
        ]
      }
    }
  }
};

export const successMincron_5448540 = {
  request: {
    query: GetInvoiceDocument,
    variables: {
      accountId: '91492',
      invoiceNumber: '5448540'
    }
  },
  result: {
    data: {
      invoice: {
        invoiceNumber: '5282411',
        status: 'Open',
        terms: 'NET 30 DAYS',
        customerPo: 'WILKESBORO JIM',
        invoiceDate: '10/06/2021',
        dueDate: '10/28/2020',
        originalAmt: '',
        openBalance: '',
        age: '',
        jobNumber: 'SHOP',
        jobName: 'SHOP',
        address: {
          streetLineOne: '911 BLUEBERRY WAY',
          streetLineTwo: '',
          streetLineThree: '',
          city: 'NORTHLAKE',
          state: 'TX',
          postalCode: '76226',
          country: ''
        },
        shipDate: '11/06/2021',
        deliveryMethod: 'Our Truck',
        subtotal: '2184.28',
        tax: '162.73',
        otherCharges: '0.0',
        paidToDate: '139.91',
        invoiceItems: {
          id: 'F02FD85F-4DCE-4EA7-8459-2DA7CFD689DC',
          brand: 'Certex USA, name:2 x 12, Nylon, Eye and Eye, Ply Sling',
          partNumber: 'SLING212,mfr:CX07-0126-12',
          thumb:
            'https://images.tradeservice.com/ProductImages/DIR100193/CERTEX_CX07-0126-12_LINE_SML.jpg',
          price: '40.00',
          qty: {
            quantityOrdered: 1,
            quantityShipped: 1
          }
        }
      }
    }
  }
};
