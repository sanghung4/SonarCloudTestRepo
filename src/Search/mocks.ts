export const PRODUCT = {
  id: 'MSC-43443',
  partNumber: '43443',
  name: 'Br 1/2 Pex X 1/2 Cop Adpt',
  manufacturerName: 'Uponor',
  manufacturerNumber: 'LF4515050',
  price: 3.541,
  stock: {
    homeBranch: {
      branchName: 'TEXARKANA',
      branchId: '1023',
      address1: null,
      address2: null,
      city: null,
      state: null,
      zip: null,
      phone: null,
      distanceToBranch: null,
      availability: 84,
      __typename: 'StoreStock'
    },
    otherBranches: [
      {
        branchName: 'ABILENE PLBG',
        branchId: '1001',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 622,
        __typename: 'StoreStock'
      },
      {
        branchName: 'AMARILLO',
        branchId: '1002',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'WICHITA FLLS',
        branchId: '1006',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 19,
        __typename: 'StoreStock'
      },
      {
        branchName: 'LUBBOCK',
        branchId: '1008',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 51,
        __typename: 'StoreStock'
      },
      {
        branchName: 'ODESSA',
        branchId: '1009',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 20,
        __typename: 'StoreStock'
      },
      {
        branchName: 'BIG SPRING',
        branchId: '1010',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 9,
        __typename: 'StoreStock'
      },
      {
        branchName: 'MIDLAND SHOW',
        branchId: '1011',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: '*KILGORE',
        branchId: '1021',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'TYLER',
        branchId: '1022',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 224,
        __typename: 'StoreStock'
      },
      {
        branchName: '*NACOGDOCHES',
        branchId: '1024',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'LONGVW PLBG',
        branchId: '1027',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 111,
        __typename: 'StoreStock'
      },
      {
        branchName: 'SAN ANGELO',
        branchId: '1029',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 82,
        __typename: 'StoreStock'
      },
      {
        branchName: 'MIDLAND PLBG',
        branchId: '1032',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'CARLSBAD',
        branchId: '1035',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'ALAMOGORDO',
        branchId: '1039',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'LAS CRUCES',
        branchId: '1040',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 26,
        __typename: 'StoreStock'
      },
      {
        branchName: 'ALBUQUERQUE',
        branchId: '1044',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 268,
        __typename: 'StoreStock'
      },
      {
        branchName: 'MCALLEN',
        branchId: '1045',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'BROWNSVILLE',
        branchId: '1055',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'HARLINGEN',
        branchId: '1056',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 49,
        __typename: 'StoreStock'
      },
      {
        branchName: 'GARDEN CITY',
        branchId: '1075',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 138,
        __typename: 'StoreStock'
      },
      {
        branchName: 'HUTCHINSON',
        branchId: '1076',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 37,
        __typename: 'StoreStock'
      },
      {
        branchName: 'LUBBOCK SHOW',
        branchId: '1079',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'LAREDO',
        branchId: '1084',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'ABILENE SHOW',
        branchId: '1085',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      },
      {
        branchName: 'EL PASO',
        branchId: '1100',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 26,
        __typename: 'StoreStock'
      },
      {
        branchName: 'SANTA FE',
        branchId: '1109',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 364,
        __typename: 'StoreStock'
      },
      {
        branchName: 'EL PASO SHOW',
        branchId: '1112',
        address1: null,
        address2: null,
        city: null,
        state: null,
        zip: null,
        phone: null,
        distanceToBranch: null,
        availability: 0,
        __typename: 'StoreStock'
      }
    ],
    __typename: 'Stock'
  },
  taxonomy: null,
  categories: [
    'Pipe & Fittings',
    'Plastic Pipe & Fittings - Supply',
    'PEX Pipe & Fittings'
  ],
  technicalDocuments: [
    {
      name: 'MFR Catalog Page',
      url: 'http://images.tradeservice.com/ATTACHMENTS/DIR100147/UPONORE00439_101.pdf',
      __typename: 'TechDoc'
    },
    {
      name: 'Technical Specification',
      url: 'http://images.tradeservice.com/ATTACHMENTS/DIR100147/UPONORE00450_1.pdf',
      __typename: 'TechDoc'
    },
    {
      name: 'Installation Sheet',
      url: 'http://images.tradeservice.com/INSTALLATIONSHEETS/DIR100005/UPONORINST00041.pdf',
      __typename: 'TechDoc'
    }
  ],
  environmentalOptions: ['Low lead compliant'],
  upc: '67337223245',
  unspsc: '40183109',
  seriesModelFigureNumber: null,
  productOverview:
    'Adapter Fitting; Type Transition, Straight; Size A 1/2"; Size B 1/2"; End Connection PEX x Female Soldered; Material C69300 Brass; Material Specification ASTM F877/F1960; Application Hot and Cold Domestic Potable Water System, Hydronic Radiant Heating and Cooling System, Residential Fire Sprinkler System; Applicable Standard NSF 14/61/372, CAN/CSA B137.5, UPC, IAPMO, IBC, IMC, IRC, IPC, NSPC, NPC, UMC, ICC ES PMG 1006/1012, HUD MR 1269, UL 1821, ULC/ORD C199P; Packaging Quantity 25 per Pack;...',
  featuresAndBenefits:
    'Lead Free; Brass Sweat Adapters Transition Uponor PEX Tubing to Copper Pipe for Use in Hot and Cold Domestic Potable Water Systems, Residential Fire Safety and Hydronic Radiant Heating and Cooling Systems; One these Adapters are Safe for Direct Burial in Soil; 25 Year Limited Warranty',
  techSpecifications: [
    {
      name: 'material',
      value: 'C69300 Brass',
      __typename: 'TechSpec'
    },
    {
      name: 'Applicable Standard',
      value:
        'NSF 14/61/372, CAN/CSA B137.5, UPC, IAPMO, IBC, IMC, IRC, IPC, NSPC, NPC, UMC, ICC ES PMG 1006/1012, HUD MR 1269, UL 1821, ULC/ORD C199P',
      __typename: 'TechSpec'
    },
    {
      name: 'Application',
      value:
        'Hot and Cold Domestic Potable Water System, Hydronic Radiant Heating and Cooling System, Residential Fire Sprinkler System',
      __typename: 'TechSpec'
    },
    {
      name: 'End Connection',
      value: 'PEX x Female Soldered',
      __typename: 'TechSpec'
    },
    {
      name: 'Material',
      value: 'C69300 Brass',
      __typename: 'TechSpec'
    },
    {
      name: 'Material Specification',
      value: 'ASTM F877/F1960',
      __typename: 'TechSpec'
    },
    {
      name: 'Size A',
      value: '1/2"',
      __typename: 'TechSpec'
    },
    {
      name: 'Size B',
      value: '1/2"',
      __typename: 'TechSpec'
    },
    {
      name: 'Type',
      value: 'Transition, Straight',
      __typename: 'TechSpec'
    }
  ],
  imageUrls: {
    thumb:
      'http://images.tradeservice.com/ProductImages/DIR100108/UPONOR_LF4512525_SML.jpg',
    small:
      'http://images.tradeservice.com/ProductImages/DIR100108/UPONOR_LF4512525_SML.jpg',
    medium:
      'http://images.tradeservice.com/ProductImages/DIR100108/UPONOR_LF4512525_MED.jpg',
    large:
      'http://images.tradeservice.com/ProductImages/DIR100107/UPONOR_LF4512525_LRG.jpg',
    __typename: 'ImageUrls'
  },
  packageDimensions: null,
  minIncrementQty: 0,
  erp: 'ECLIPSE',
  __typename: 'Product'
};
