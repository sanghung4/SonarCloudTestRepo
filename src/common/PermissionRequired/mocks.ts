import { UserContextType } from 'AuthProvider';
import { Permission } from 'common/PermissionRequired';

export const authConfigNoPerm = {
  authState: {
    isAuthenticated: true
  },
  profile: {
    isEmployee: true,
    isVerified: true
  } as UserContextType['profile'],
  activeFeatures: ['LISTS']
};
export const authConfig = {
  authState: {
    isAuthenticated: true
  },
  profile: {
    userId: 'test',
    permissions: [Permission.EDIT_LIST, Permission.APPROVE_CART],
    isEmployee: true,
    isVerified: true
  } as UserContextType['profile'],
  activeFeatures: ['LISTS']
};
