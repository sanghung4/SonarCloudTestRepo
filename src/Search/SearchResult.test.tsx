import { BranchContext } from 'providers/BranchProvider';
import SearchResult from 'Search/SearchResult';
import { setBreakpoint } from 'test-utils/mockMediaQuery';
import { PRICING_DATA, PRODUCT } from 'Search/SearchResult.mocks';
import { render } from 'test-utils/TestWrapper';

describe('SearchResult tests', () => {
  it('should render correctly on Desktop', () => {
    setBreakpoint('desktop');
    const { container } = render(
      <SearchResult
        product={PRODUCT}
        pricing={PRICING_DATA}
        pricingLoading={false}
        index={0}
        onClick={() => {}}
      />
    );
    expect(container).toMatchSnapshot();
  });

  it('should render correctly on mobile', () => {
    setBreakpoint('mobile');
    const { container } = render(
      <SearchResult
        product={PRODUCT}
        pricing={PRICING_DATA}
        pricingLoading={false}
        index={0}
        onClick={() => {}}
      />
    );
    expect(container).toMatchSnapshot();
  });

  it('should display the product image', async () => {
    const { findAllByRole } = render(
      <SearchResult
        product={PRODUCT}
        pricing={PRICING_DATA}
        pricingLoading={false}
        index={0}
        onClick={() => {}}
      />
    );
    const productImage = await findAllByRole('img');
    expect(productImage[0]).toBeInTheDocument();
    expect(productImage[0]).toMatchSnapshot();
  });
});
