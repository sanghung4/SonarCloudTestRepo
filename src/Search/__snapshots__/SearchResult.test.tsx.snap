// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResult tests should display the product image 1`] = `
<img
  alt="Br 1/2 Pex X 1/2 Cop Adpt"
  class="MuiBox-root css-4w7ia0"
  src="http://images.tradeservice.com/ProductImages/DIR100108/UPONOR_LF4512525_SML.jpg"
  title="Br 1/2 Pex X 1/2 Cop Adpt"
/>
`;

exports[`SearchResult tests should render correctly on Desktop 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-fsi2oy-MuiPaper-root-MuiCard-root"
    data-testid="search-result-0"
  >
    <div
      class="MuiCardContent-root css-145c6cf-MuiCardContent-root"
    >
      <div
        class="MuiBox-root css-1u8pvn2"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-disableElevation MuiButtonBase-root  css-wyox7y-MuiButtonBase-root-MuiButton-root"
          data-testid="add-to-list-button-0"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeMedium css-1sx3uk1-MuiButton-startIcon"
          >
            <svg>
              star-empty.svg
            </svg>
          </span>
          Add to List
        </button>
      </div>
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
        href="/product/uponor-br-1-2-pex-x-1-2-cop-adpt-lf4515050/MSC-43443"
      >
        <div
          class="MuiBox-root css-6iudor"
          data-testid="product-image-0"
        >
          <img
            alt="Br 1/2 Pex X 1/2 Cop Adpt"
            class="MuiBox-root css-4w7ia0"
            src="http://images.tradeservice.com/ProductImages/DIR100108/UPONOR_LF4512525_SML.jpg"
            title="Br 1/2 Pex X 1/2 Cop Adpt"
          />
        </div>
      </a>
      <p
        class="MuiTypography-root MuiTypography-caption MuiTypography-gutterBottom css-slmvot-MuiTypography-root"
        data-testid="product-manufacturer-name-0"
      >
        Uponor
      </p>
      <div
        class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-19m3wr6-MuiTypography-root"
      >
        <div>
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
            data-testid="product-name-0"
            href="/product/uponor-br-1-2-pex-x-1-2-cop-adpt-lf4515050/MSC-43443"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-12fo3br-MuiTypography-root"
            >
              Br 1/2 Pex X 1/2 Cop Adpt
            </p>
          </a>
        </div>
      </div>
      <p
        class="MuiTypography-root MuiTypography-caption css-12u2emh-MuiTypography-root"
        data-testid="product-manufacturer-number-0"
      >
        MFR#
         
        LF4515050
      </p>
    </div>
    <div
      class="MuiBox-root css-eskpy5"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-1oiwmw4-MuiTypography-root"
        data-testid="product-price-0"
      >
        <span
          class="MuiTypography-root MuiTypography-h4 css-te0q99-MuiTypography-root"
          data-testid="product-sell-price-0"
        >
          $3.54
           
        </span>
        <span
          class="MuiTypography-root MuiTypography-body1 css-v0l0ba-MuiTypography-root"
          data-testid="product-each-0"
        >
          ea
        </span>
      </h5>
      <div
        class="MuiBox-root css-swdnp0"
        data-testid="availability-stock-chip-0"
        title="84 in stock"
      >
        <svg>
          in-stock.svg
        </svg>
         84 in stock
      </div>
    </div>
    <div
      class="MuiCardActions-root MuiCardActions-spacing css-1i30rla-MuiCardActions-root"
    >
      <div
        class="number-input__container"
      >
        <div
          class="number-input__button-container extra-small"
        >
          <button
            class="number-input__button"
            data-testid="quantity-input-down-0"
            disabled=""
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="RemoveIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13H5v-2h14v2z"
              />
            </svg>
          </button>
        </div>
        <input
          class="number-input__input extra-small"
          data-testid="quantity-input-0"
          disabled=""
          index="0"
          min="1"
          style="height: 30px;"
          value="1"
        />
        <div
          class="number-input__button-container extra-small"
        >
          <button
            class="number-input__button"
            data-testid="quantity-input-up-0"
            disabled=""
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="AddIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        aria-label=""
        class=""
        data-mui-internal-clone-element="true"
        data-testid="tipDesktop"
      >
        <button
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-disableElevation MuiButton-fullWidth MuiButtonBase-root Mui-disabled  css-13bu29b-MuiButtonBase-root-MuiButton-root"
          data-testid="add-to-cart-button-0"
          disabled=""
          tabindex="-1"
          type="button"
        >
          Add to Cart
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`SearchResult tests should render correctly on mobile 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
    data-testid="search-result-0"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-c6onk3-MuiGrid-root"
    >
      <button
        class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-disableElevation MuiButtonBase-root  css-wyox7y-MuiButtonBase-root-MuiButton-root"
        data-testid="add-to-list-button-0"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-startIcon MuiButton-iconSizeMedium css-1sx3uk1-MuiButton-startIcon"
        >
          <svg>
            star-empty.svg
          </svg>
        </span>
        Add to List
      </button>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-capmjd-MuiGrid-root"
    >
      <div
        class="MuiBox-root css-1vce2kz"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
          href="/product/uponor-br-1-2-pex-x-1-2-cop-adpt-lf4515050/MSC-43443"
        >
          <div
            class="MuiBox-root css-0"
            data-testid="product-image-0"
          >
            <img
              alt="Br 1/2 Pex X 1/2 Cop Adpt"
              class="MuiBox-root css-4w7ia0"
              src="http://images.tradeservice.com/ProductImages/DIR100108/UPONOR_LF4512525_SML.jpg"
              title="Br 1/2 Pex X 1/2 Cop Adpt"
            />
          </div>
        </a>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-1mum6ye-MuiGrid-root"
    >
      <div
        class="MuiBox-root css-1rr4qq7"
      >
        <p
          class="MuiTypography-root MuiTypography-caption MuiTypography-gutterBottom css-slmvot-MuiTypography-root"
          data-testid="product-manufacturer-name-0"
        >
          Uponor
        </p>
        <div
          class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-19m3wr6-MuiTypography-root"
          data-testid="product-name-0"
        >
          <div>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
              href="/product/uponor-br-1-2-pex-x-1-2-cop-adpt-lf4515050/MSC-43443"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-12fo3br-MuiTypography-root"
              >
                Br 1/2 Pex X 1/2 Cop Adpt
              </p>
            </a>
          </div>
        </div>
        <div
          class="MuiBox-root css-1qm1lh"
        >
          <p
            class="MuiTypography-root MuiTypography-caption css-12u2emh-MuiTypography-root"
            data-testid="product-manufacturer-number-0"
          >
            MFR#
             
            LF4515050
          </p>
        </div>
        <div
          class="MuiBox-root css-19unpuq"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-1eebjzl-MuiTypography-root"
            data-testid="product-price-0"
          >
            <span
              class="MuiTypography-root MuiTypography-h4 css-te0q99-MuiTypography-root"
              data-testid="product-sell-price-0"
            >
              $3.54
               
            </span>
            <span
              class="MuiTypography-root MuiTypography-body1 css-1faez38-MuiTypography-root"
              data-testid="product-each-0"
            >
              ea
            </span>
          </h5>
          <div
            class="MuiBox-root css-swdnp0"
            data-testid="availability-stock-chip"
            title="84 in stock"
          >
            <svg>
              in-stock.svg
            </svg>
             84 in stock
          </div>
        </div>
        <div
          class="MuiBox-root css-7pf6at"
        >
          <div
            class="number-input__container full-width"
          >
            <div
              class="number-input__button-container small"
            >
              <button
                class="number-input__button"
                data-testid="quantity-input-down-0"
                disabled=""
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="RemoveIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </button>
            </div>
            <input
              class="number-input__input small"
              data-testid="quantity-input-0"
              disabled=""
              index="0"
              min="1"
              value="1"
            />
            <div
              class="number-input__button-container small"
            >
              <button
                class="number-input__button"
                data-testid="quantity-input-up-0"
                disabled=""
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="AddIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </button>
            </div>
          </div>
          <button
            class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButton-fullWidth MuiButtonBase-root Mui-disabled  css-v09etv-MuiButtonBase-root-MuiButton-root"
            data-testid="add-to-cart-button-0"
            disabled=""
            tabindex="-1"
            type="button"
          >
            Add to Cart
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
